api-report/container-loader.api.md

```diff
@@ -125,6 +125,8 @@ export interface IProtocolHandler extends IProtocolHandler_2 {
     readonly audience: IAudienceOwner;
     // (undocumented)
     processSignal(message: ISignalMessage): any;
+    // (undocumented)
+    shouldProcessSignal?(message: ISignalMessage): any;
```

Comment:
why put this on the interface at all, couldn't it just be a free function? do we see a need right now for protocol handlers to control this. In general, we shouldn't expose things that aren't needed

Reply:
That's a great suggestion! Done! I really didn't want to deal with the headache of a new version!